public/data/taravis.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "Anu Wat",
+  "bio": "Web Dev, Interactive Media Dev",
+  "avatar": "https://github.com/anuwatavis.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/anuwatavis.png",
```

```suggestion
      "url": "https://github.com/anuwatavis",
```
By mistake you have add avatar link